workspaces/rstudio/Dockerfile

```diff
@@ -62,6 +62,7 @@ RUN export OLD_UID=$(id -u rstudio) && \
 USER 0
 RUN apt-get update && \
     # Run 'unminimize' to add docs
+    apt-get install -y man-db unminimize && \
```

Comment:
Related:

- https://github.com/PrairieLearn/PrairieLearn/pull/10728#issuecomment-2405897726
- https://github.com/PrairieLearn/PrairieLearn/pull/10734#discussion_r1796209122

Reply:
Can you expand?  Those two changes do not seem R related.

Reply:
`rocker/rstudio:4.4.2` is now based on Ubuntu 24, which does not include the `unminimize` command by default. We faced this during two other image upgrades (Ubuntu 2X -> 24) and had to make similar changes (explicitly install `unminimize`).

Reply:
Ah, thanks.  30 years of Debian and Ubuntu here and "TIL" about `unminimize`.  From a glance at `apt-cache show unminimize` it seems cloud / container related ... which may explain it.  

All good then.